src/store/getters.ts

```diff
@@ -312,8 +317,19 @@ export default {
     return `${name} ${enDash} ${date}`
   },
 
-  /** */
+  /**
+   * Is True if business is in Limited Restoration state, ie
+   * the last filing to change state was a Restoration filing.
+   */
   isEntityInLimitedRestoration (state: StateIF): boolean {
-    return (state.stateFiling?.business?.state === 'ACTIVE' && state.stateFiling.hasOwnProperty('restoration'))
+    return (state.stateFiling?.hasOwnProperty(FilingTypes.RESTORATION))
+  },
```

Comment:
It's redundant to check if the business is currently active -- it must be if the last state filing was a restoration. (If a limited restoration expires then there will be a new state filing to re-dissolve the business.)

You could argue that checking current business state is a safe thing to do, but I think we're more likely to catch state issues if we don't add that check than if we do (ie, we might notice a state mismatch but we might not notice a missing badge or menu item).